packages/cli/src/util/login/bitbucket.ts

```diff
@@ -0,0 +1,11 @@
+import { URL } from 'url';
+import { LoginParams } from './types';
+import doOauthLogin from './oauth';
+
+export default function doBitbucketLogin(params: LoginParams) {
+  const url = new URL(
+    '/api/registration/bitbucket/connect',
+    'https://vercel.com'
```

Comment:
Lets leave a comment here mentioning why we cant use `apiUrl` like we do for others